tests/test_content_manifest.py

```diff
@@ -153,28 +153,36 @@ def test_process_pip(default_request, default_toplevel_purl):
 
 
 @pytest.mark.parametrize(
-    "package", [None, {"name": "example.com/org/project", "type": "go-package", "version": "1.1.1"}]
+    "packages",
+    [
+        [
+            None,
+            {"name": "example.com/org/project", "type": "go-package", "version": "1.1.1"},
+            {"name": "example.com/org/project", "type": "git-submodule", "version": "1.1.1"},
+        ],
+    ],
 )
-def test_to_json(app, package):
+@mock.patch("cachito.web.content_manifest.ContentManifest.generate_icm")
+def test_to_json(mock_generate_icm, app, packages):
```

Comment:
We should also have a test for processing git submodules similar to `test_process_{go,npm,pip}`